currencies/src/main/java/com/generalbytes/batm/common/currencies/CryptoCurrency.java

```diff
@@ -77,6 +77,7 @@
     POT("PotCoin"),
     REP("Augur"),
     SMART("SmartCash"),
+    AQUANOW("Aquanow"),
```

Comment:
You declare this currency, but you don't use it anywhere.

Reply:
Removed. We don't have any cryptocurrency.